include/MySQL_Monitor.hpp

```diff
@@ -1,10 +1,16 @@
 #ifndef __CLASS_MYSQL_MONITOR_H
 #define __CLASS_MYSQL_MONITOR_H
+
+#include "MySQL_HostGroups_Manager.h"
 #include "proxysql.h"
 #include "cpp.h"
 #include "thread.h"
 #include "wqueue.h"
 
+// Headers for declaring Prometheus counters
+#include <prometheus/counter.h>
+#include <prometheus/gauge.h>
```

Comment:
Please use quotes and not brackets .
https://gcc.gnu.org/onlinedocs/cpp/Search-Path.html

Reply:
Solved!